### ISSUE TITLE
makefont.c : "Multiple --texture parameters"

### ISSUE THREAD

**Issue by sasmaster** [rougier/freetype-gl#187]
Hi. I am trying to run makefont to generate sdf texture. My cmd arguments look like this:

**--font fonts/arial.ttf --header arial.h  --size  32 --texture 512 --rendermode 'sdf'**

But I am getting  "Multiple --texture parameters"

For some reason the code compares my texture size to the default (128.0) inside --rendermode loop and exists with this strange error. Am I missing anything in my cmd arguments?

**Comment by rougier** [comment]
Just checked the source and I don't quite understand either. It seems you cannot have a texture size != 128 and this is quite weird.

**Comment by gustafsson** [comment]
Nope, not by me. But it seems @adrianbroher copied the block for parsing `--texture` arguments when he created the block for parsing `--rendermode` arguments here: https://github.com/rougier/freetype-gl/commit/75a05c4db4ea6a6db70f30d7a81d2a5626689ebf

**Comment by sasmaster** [comment]
I tested without that check and it works fine. Looks like someone forgot to
remove it  ))

**Comment by rougier** [comment]
@gustafsson Sorry
@sasmaster an you make a PR?

**Comment by sasmaster** [comment]
I will.Need to fork this project first. because currently I use your code in a completely different project

https://github.com/sasmaster/FMG

**Comment by sasmaster** [comment]
Btw,I forgot to mention it, if I put the commands in this order:

>    --font "fonts/arial.ttf" --header "data/arial.h"  --size  32  --rendermode "sdf" --texture  512

So that --texture is put last, then the cmd passes ok.

Anyway,I guess that chunk should be removed from the second place.

**Comment by rougier** [comment]
Nice project indeed

**Comment by Mark-Weston** [comment]
--texture must be last argument

**Comment by einsteinx2** [comment]
Fixed this here in this PR along with a few other improvements: https://github.com/rougier/freetype-gl/pull/213
